try{
    console.log(a);
    var a = 10;
    // let a = 10;
}
catch{
    console.log("Variable a is initialised with let hence it throws error");
}
finally{
    console.log("Always");
}